frontend/apps/reader/modules/readerzooming.lua

```diff
@@ -280,6 +280,26 @@ function ReaderZooming:onDefineZoomMode(btn)
         self.ui:handleEvent(Event:new("SetZoomPan", settings, true))
     end
     self.ui:handleEvent(Event:new("SetZoomMode", zoom_mode))
+    if tonumber(btn) then
+        UIManager:show(InfoMessage:new{
+            timeout = 1.5,
```

Comment:
Please use 2 for consistency with larger info popups in the rest of the program (or change the rest to 1.5, fine by me too but that'd be best as a separate PR).